pkg/parser/parser.y

```diff
@@ -13770,8 +13770,6 @@ ConnectionOptions:
 |	"WITH" ConnectionOptionList
 	{
 		$$ = $2
-		yylex.AppendError(yylex.Errorf("TiDB does not support WITH ConnectionOptions now, they would be parsed but ignored."))
-		parser.lastErrorAsWarn()
```

Comment:
we only support `MAX_USER_CONNECTIONS`, maybe keep reporting error for all others?

Reply:
> we only support `MAX_USER_CONNECTIONS`, maybe keep reporting error for all others?

Good comments. I have modify it. If enter other `WITH xxx` but `WITH MAX_USER_CONNECTIONS`, it wiil report warning.